metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -145,6 +145,26 @@ public void run() {
     }
 
     /**
+     * Starts the reporter polling at the given period.
```

Comment:
Could you add a simple unit test to `ScheduledReporterTest`? A simple test which verifies that the scheduler wakes up with a correct initial delay. Very similar to the `pollsPeriodically` test.